src/docfx/build/toc/TableOfContentsLoader.cs

```diff
@@ -192,6 +192,17 @@ private void TraverseAndConvertHref(TableOfContentsNode node, JoinTOCConfig join
                 // Resolve
                 node.Items = LoadTocNodes(node.Items, file, rootPath, referencedFiles, referencedTocs);
 
+                if (file.Path.Value.Contains("_splitted/"))
+                {
+                    foreach (var item in node.Items)
+                    {
+                        if (item.Value != null)
+                        {
+                            item.Value.Monikers = _monikerProvider.GetFileLevelMonikers(_errors, item.Value?.Document ?? file);
```

Comment:
is it possible to set monikers during split toc?
https://github.com/dotnet/docfx/blob/a255e4aba4fb5728d5ef7346de00fb72ba000625/src/docfx/build/toc/TableOfContentsMap.cs#L256

Reply:
I recalled that adding monikers here cannot resolve the problem because I have tried adding monikers here. Will give you more information about this.